src/main/java/duke/command/EventCommand.java

```diff
@@ -0,0 +1,52 @@
+package duke.command;
+
+import duke.data.TaskList;
+import duke.data.task.Task;
+import duke.exception.DukeException;
+import duke.exception.EventMissingDescriptionException;
+import duke.exception.EventMissingTagException;
+import duke.exception.EventMissingTimeException;
+import duke.storage.Storage;
+import duke.ui.Ui;
+
+public class EventCommand extends AddCommand {
+    public static final String COMMAND_WORD = "EVENT";
+    private String eventTime;
+
+    public EventCommand() {
+        super();
+    }
+
+    @Override
+    public void checkAndSetParameters(String parameterInput) throws DukeException {
+        checkParameters(parameterInput);
+        setParameters(parameterInput);
+    }
+
+    @Override
+    protected void checkParameters(String parameterInput) throws DukeException {
+        if (!containsTimeSeparator(parameterInput, " /at ")) {
```

Comment:
Same as comment for `" /by "`.

Reply:
Now added as a constant of the class:
`    public static final String TIME_SEPARATOR = " /at ";`